src/js/views/maps/ViewFinderView.js

```diff
@@ -0,0 +1,155 @@
+"use strict";
+
+define([
+  "backbone",
+  "text!templates/maps/view-finder.html",
+], (
+  Backbone,
+  Template,
+) => {
+  /**
+   * @class ViewFinderView
+   * @classdesc The ViewFinderView allows a user to search for a latitude and longitude in the map view.
+   * @classcategory Views/Maps
+   * @name ViewFinderView
+   * @extends Backbone.View
+   * @screenshot views/maps/ViewFinderView.png
+   * @since 2.27.1
+   * @constructs ViewFinderView
+   */
+  var ViewFinderView = Backbone.View.extend({
+    /**
+     * The type of View this is
+     * @type {string}
+     */
+    type: "ViewFinderView",
+
+    /**
+     * The HTML classes to use for this view's element
+     * @type {string}
+     */
+    className: classNames.baseClass,
+
+    /**
+    * The events this view will listen to and the associated function to call.
+    * @type {Object}
+    */
+    events: {
+      [`click .${classNames.button}`]: 'search',
+      [`keyup .${classNames.input}`]: 'keyup',
+    },
+
+    /** 
+     * Values meant to be used by the rendered HTML template.
+     */
+    templateVars: {
+      errorMessage: "",
+      inputValue: "",
+      placeholder: "Search by latitude and longitude",
+      classNames,
+    },
+
+    /**
+     * @typedef {Object} ViewFinderViewOptions
+     * @property {Map} The Map model associated with this view allowing control
+     * of panning to different locations on the map. 
+     */
+    initialize(options) {
+      this.mapModel = options.model;
+    },
+
+    render() {
+      this.focusInput();
+
+      this.el.innerHTML = _.template(Template)(this.templateVars);
+    },
+
+    /**
+     * Focus the input field on a delay to allow for the input to be
+     * visible on the page before attempting to focus.
+     */
+    focusInput() {
+      _.defer(() => {
```

Comment:
Just curious- what's the error case you ran into here that prompted _.defer?

Reply:
Does calling focusInput() after `this.el.innerHTML =` work?

Reply:
Yep! Good call